nbbuild/antsrc/org/netbeans/nbbuild/CustomJavac.java

```diff
@@ -135,9 +135,10 @@ protected void compile() {
         super.compile();
     }
 
-    private boolean isBootclasspathOptionUsed() {
+    private boolean canUseRelease() {
```

Comment:
The logic and the name of this method seem to be opposite?

Reply:
oops. good catch. will flip it

Reply:
fixed - thanks for pointing this out.